packages/frontend/src/pages/Tile/components/TableBanner/TableCollaborators.tsx

```diff
@@ -73,7 +73,7 @@ const AddNewCollaborator = ({
   const onSubmit: FormEventHandler<HTMLFormElement> = useCallback(
     async (e) => {
       try {
-        e.preventDefault()
+        e.stopPropagation()
```

Comment:
question: can i ask why you change to `stopPropagation` ah

Reply:
oops i forgot to change back